grasa_event_locator/views.py

```diff
@@ -141,7 +142,7 @@ def changepw(request):
                             request.user.set_password(new)
                             request.user.save()
                     else:
-                            print("No")
+                            return()
```

Comment:
`return` is a keyword, not a method. So this should be:

```suggestion
                            return
```

Reply:
Copy that! Suggestion committed.